packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -241,6 +241,7 @@ export default class Calendar extends React.Component {
       date: this.localizeDate(this.getDateInView()),
       selectingDate: null,
       calendarIsKeyboardFocused: false,
+      disableKeyboardSelected: false,
```

Comment:
This issue is reproducible sans keyboard - via mouse clicks. How does this fix the issue?

Reply:
Reolved in 15987a0